src/app/static/static-values.ts

```diff
@@ -164,10 +160,6 @@ export class StaticValues {
     'SUSHI_ETH_UST',
     // 'SUSHI_MIC_USDT',
     // 'SUSHI_MIS_USDT',
-    'ONEINCH_ETH_DAI',
-    'ONEINCH_ETH_USDC',
```

Comment:
do we have any strategies instead of it?

Reply:
```
    'ETH_DAI_HODL',
    'ETH_USDC_HODL',
    'ETH_USDT_HODL',
    'ETH_WBTC_HODL',
```

are the new ones